draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -145,19 +159,24 @@ Selective disclosure:
 :  Process of a Holder disclosing to a Verifier a subset of claims contained in a claim set issued by an Issuer.
 
 Selectively Disclosable JWT (SD-JWT):
-:  A composite structure, consisting of an Issuer-signed JWT (JWS, [@!RFC7515]), Disclosures, and optionally a Key Binding JWT
- that supports selective disclosure as defined in this document. It can contain both regular claims and digests of selectively-disclosable claims.
+:  A composite structure, consisting of an Issuer-signed JWT (JWS, [@!RFC7515]) and zero or more Disclosures, which
+   supports selective disclosure as defined in this document. It can contain both regular claims and digests of selectively-disclosable claims.
 
 Disclosure:
 :  A combination of a salt, a cleartext claim name (present when the claim is a name-value pair and absent when the claim is an array element), and a cleartext claim value, all of which are used to calculate a digest for the respective claim.
 
 Key Binding:
 :  Ability of the Holder to prove legitimate possession of an SD-JWT by proving
-  control over the same private key during the issuance and presentation. An SD-JWT with Key Binding contains
-  a public key, or a reference to a public key, that matches to the private key controlled by the Holder.
+  control over the same private key during the issuance and presentation. To be compatible with Key Binding, an SD-JWT MUST contain
```

Comment:
```suggestion
  control over the same private key during the issuance and presentation. To be compatible with Key Binding, an SD-JWT needs to contain
```
not sure it is appropriate to have MUST in a term definition?

Reply:
```suggestion
  control over the same private key during the issuance and presentation. To be compatible with Key Binding, an SD-JWT contains
```